### PR TITLE
Add events table and implement assorted database changes

### PR DESCRIPTION
**Pull request by TomCusack-Softwire**
Implements events and parcel information I forgot about 💀

### CODE REVIEW

File: src/database_types_file.ts [hunk 1]
```diff
@@ -22,11 +22,13 @@ export interface Database {
           extra_information: string
           family_id: string
           feminine_products: string[]
+          flag_attention: boolean
           full_name: string
           other_items: string[]
           pet_food: string[]
           phone_number: string
           primary_key: string
+          signposting_call: boolean
```
Comment (daniel-conlan-softwire) [inline src/database_types_file.ts:31]: Should the fields `flag_attention` and `signposting_call` not be for each individual parcel? Also, would field names such as `flagged_for_attention` or `signposting_call_required` be a bit more clear?

File: src/database_types_file.ts [hunk 2]
```diff
@@ -58,14 +62,44 @@ export interface Database {
...
           pet_food?: string[]
           phone_number?: string
           primary_key?: string
+          signposting_call?: boolean
         }
         Relationships: []
       }
+      events: {
+        Row: {
+          event_name: string
```
Comment (daniel-conlan-softwire) [inline src/database_types_file.ts:77]: I presume `event_name` is the description of a parcel event - for example, "Shopping List Printed"?

**Review by daniel-conlan-softwire — APPROVED** [review]
Thanks for this, looks all good

File: src/database_types_file.ts (hunk 1 above)
Comment (TomCusack-Softwire) [reply src/database_types_file.ts:31]: sure will change

**Pull request merged**